fix(versions): update Activiti/activiti-cloud-runtime-bundle-service versions into develop

jx-activiti-cloud opened the pull request:
[UpdateBot](https://github.com/jenkins-x/updatebot) pushed maven dependency: `org.activiti.dependencies:activiti-dependencies` to: `7.1.58`

erdemedeiros committed:
Add -ifNotExists flag

Necessary from h2 1.4.98 (Spring boot 2.1.6.RELASE uses 1.4.99)
Ref https://github.com/Activiti/Activiti/issues/2616

Commit:
fix(version): update org.activiti.dependencies:activiti-dependencies to 7.1.58

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.dependencies:activiti-dependencies 7.1.58 org.activiti.api:activiti-api-dependencies 7.1.45 org.activiti.core.common:activiti-core-common-dependencies 7.1.41 org.activiti.build:activiti-parent 7.1.27

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.cloud.build:activiti-cloud-parent 7.1.10 org.activiti.cloud.build:activiti-cloud-dependencies-parent 7.1.10

Commit:
fix(version): update org.activiti.cloud.build:activiti-cloud-parent to 7.1.10

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.dependencies:activiti-dependencies 7.1.59 org.activiti.api:activiti-api-dependencies 7.1.46 org.activiti.core.common:activiti-core-common-dependencies 7.1.42 org.activiti.build:activiti-parent 7.1.28

Commit:
fix(version): update org.activiti.dependencies:activiti-dependencies to 7.1.59

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.cloud.common:activiti-cloud-service-common-dependencies 7.1.55

Commit:
fix(version): update org.activiti.cloud.common:activiti-cloud-service-common-dependencies to 7.1.55

igdianov committed:
fix: No qualifying bean of type 'org.activiti.engine.ManagementService'

igdianov committed:
fix: managementService bean to TestProcessEngineConfiguration

igdianov committed:
fix: add managementService mock bean to CloudConnectorsAutoConfigurationIT

igdianov committed:
fix: add repositoryService mock bean to ProcessInstanceVariableAdminControllerImplIT

igdianov committed:
fix: Actual and expected have the same elements but not in the same order 

shouldExecuteProcessWithEmbeddedSubProcessContainingSignalIntermediateCatchEvent(org.activiti.cloud.starter.tests.services.audit.EmbeddedSubProcessAuditIT)  Time elapsed: 10.148 s  <<< ERROR!
org.awaitility.core.ConditionTimeoutException: 
Assertion condition defined as a lambda expression in org.activiti.cloud.starter.tests.services.audit.EmbeddedSubProcessAuditIT 
Actual and expected have the same elements but not in the same order, at index 7 actual element was:
  <(ACTIVITY_STARTED, "a9cd5e8d-d34a-11e9-af70-0242df045af3", null, "signalSubProcess")>
whereas expected element was:
  <(TASK_CREATED, "a9cd5e8d-d34a-11e9-af70-0242df045af3", null, "signalSubProcess")>
 within 10 seconds.
	at org.activiti.cloud.starter.tests.services.audit.EmbeddedSubProcessAuditIT.shouldExecuteProcessWithEmbeddedSubProcessContainingSignalIntermediateCatchEvent(EmbeddedSubProcessAuditIT.java:353)
Caused by: java.lang.AssertionError: 
Actual and expected have the same elements but not in the same order, at index 7 actual element was:
  <(ACTIVITY_STARTED, "a9cd5e8d-d34a-11e9-af70-0242df045af3", null, "signalSubProcess")>
whereas expected element was:
  <(TASK_CREATED, "a9cd5e8d-d34a-11e9-af70-0242df045af3", null, "signalSubProcess")>
	at org.activiti.cloud.starter.tests.services.audit.EmbeddedSubProcessAuditIT.lambda$shouldExecuteProcessWithEmbeddedSubProcessContainingSignalIntermediateCatchEvent$17(EmbeddedSubProcessAuditIT.java:362)

igdianov committed:
fix: testTwoSubProcesses(AuditProducerIT)

Assertion condition defined as a lambda expression in org.activiti.cloud.starter.tests.services.audit.AuditProducerIT 
Actual and expected have the same elements but not in the same order, at index 11 actual element was:
  <(ACTIVITY_STARTED, "be3069f8-d357-11e9-9f87-024210a72532", null, "callTwoSubProcesses")>
whereas expected element was:
  <(PROCESS_STARTED, "be31c98c-d357-11e9-9f87-024210a72532", "be3069f8-d357-11e9-9f87-024210a72532", "simpleSubProcess1")>
 within 10 seconds.
	at org.activiti.cloud.starter.tests.services.audit.AuditProducerIT.testTwoSubProcesses(AuditProducerIT.java:605)
Caused by: java.lang.AssertionError: 
Actual and expected have the same elements but not in the same order, at index 11 actual element was:
  <(ACTIVITY_STARTED, "be3069f8-d357-11e9-9f87-024210a72532", null, "callTwoSubProcesses")>
whereas expected element was:
  <(PROCESS_STARTED, "be31c98c-d357-11e9-9f87-024210a72532", "be3069f8-d357-11e9-9f87-024210a72532", "simpleSubProcess1")>
	at org.activiti.cloud.starter.tests.services.audit.AuditProducerIT.lambda$testTwoSubProcesses$46(AuditProducerIT.java:613)

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.dependencies:activiti-dependencies 7.1.60 org.activiti.api:activiti-api-dependencies 7.1.46 org.activiti.core.common:activiti-core-common-dependencies 7.1.42 org.activiti.build:activiti-parent 7.1.28

Commit:
fix(version): update org.activiti.dependencies:activiti-dependencies to 7.1.60

erdemedeiros committed:
Remove wait forever
